### PR TITLE
rm rent check where unnecessary

### PR DESCRIPTION
**Pull request by segfaultdoc**

### CODE REVIEW

File: tip-payment/programs/tip-distribution/src/state.rs [hunk 1]
```diff
@@ -121,43 +119,28 @@ impl TipDistributionAccount {
...
 
-    /// Transfers funds from-from and to-to. Returns an error if from ends up with less than what's
-    /// required for rent exemption.
-    fn checked_transfer(
-        from: AccountInfo,
-        to: AccountInfo,
-        amount: u64,
-        min_rent_lamports: u64,
-    ) -> Result<()> {
+    fn transfer_lamports(from: AccountInfo, to: AccountInfo, amount: u64) -> Result<()> {
```
Comment (esemeniuc) [inline tip-payment/programs/tip-distribution/src/state.rs:134]: will solana runtime revert if lamports < min rent exempt?

File: update_merkle_root_authority/Cargo.toml [hunk 2]
```diff
@@ -14,9 +14,9 @@ log = "0.4.17"
...
 serde_json = "1.0.81"
-solana-client = "~1.10.29"
-solana-program = "~1.10.29"
-solana-sdk = "~1.10.29"
+solana-client = "1.14.13"
+solana-program = "1.14.13"
+solana-sdk = "1.14.13"
 thiserror = "1.0.31"
 tip-distribution = { path = "../tip-payment/programs/tip-distribution", features = ["no-entrypoint"] }
 tokio = { version = "1.12.0", features = ["rt-multi-thread", "macros", "sync", "time", "full"] }
```
Comment (esemeniuc) [inline update_merkle_root_authority/Cargo.toml:22]: maybe bump to `tokio = "~1.14.1"` to match other apps?

**Review by esemeniuc — APPROVED** [review]
LGTM

File: tip-payment/programs/tip-distribution/src/state.rs (hunk 1 above)
Comment (segfaultdoc) [reply tip-payment/programs/tip-distribution/src/state.rs:134]: yep

**Pull request merged**